config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
   namespace :api do
     namespace :v1 do
       resources :events, only: %i[index create show update destroy]
+      resources :users, only: %i[index]
```

Comment:
Here you can modify resources to have custom controller action
```suggestion
      resources :users, only: %i[index] do
          get :curent_user_details
      end
```
To see what path it create run `bundle exec rails routes` command

Reply:
Resources were modified and custom action was added.